index.js

```diff
@@ -13,9 +13,9 @@ try {
  * Defer.
  */
 
-var defer = typeof process === 'undefined' || typeof process !== 'function'
-  ? function(fn){ setTimeout(fn); }
-  : process.nextTick;
+var defer = typeof process !== 'undefined' && typeof process.nextTick === 'function'
```

Comment:
if you want, i can update the line to explicitly check against `process === null`. can this land if i do?

```
var defer = typeof process !== 'undefined' && process !== null && typeof process.nextTick === 'function'
  ? process.nextTick
  : function(fn){ setTimeout(fn); };
```